build/make/build.mk

```diff
@@ -1,3 +1,5 @@
+##@ Building go software
```

Comment:
I don't know if `Building` is too generic... Maybe `Compiling with go` or something?